crates/fuel-core/src/database.rs

```diff
@@ -207,6 +207,17 @@ where
         state_rewind_policy: StateRewindPolicy,
         database_config: DatabaseConfig,
     ) -> Result<Self> {
+        let db = Self::open_as_rocksdb(path, state_rewind_policy, database_config)?;
+
+        Ok(Self::new(Arc::new(db)))
+    }
+
+    #[cfg(feature = "rocksdb")]
+    pub fn open_as_rocksdb(
```

Comment:
The name make it hard to differentiate with `open_rocksdb`

Reply:
okay

Reply:
addressed in [69412f6](https://github.com/FuelLabs/fuel-core/pull/2532/commits/69412f62a73d6624069c60801e44c60b5cbad0fb)